db/seeders/20220918075630-seed-data.js

```diff
@@ -3,70 +3,76 @@ module.exports = {
     // Define user data
     const userData = [
       {
-        first_name: 'John',
-        last_name: 'Doe',
-        username: 'some username',
-        address: 'some address',
-        phone: 'some phone number',
-        email: 'john@gmail.com',
-        password: 'ec5fdc07d43151d336f4f7ef520d1f3f590896be38d2833b9546a4af9b9110681f0267b2e2f3b7dd5fe1bd46514f874fbb6059c3386832ac26e3d99dfc6b0dcf',
```

Comment:
@Nyx92 I'm not really sure what this hash is, but when I typed `password` it didn't work so I had to change the hashes